tempesta_fw/http.c

```diff
@@ -2104,6 +2176,11 @@ static void
 tfw_http_cli_error_resp_and_log(bool reply, bool nolog, TfwHttpReq *req,
 				int status, const char *msg)
 {
+	if (req->flags & TFW_HTTP_HMONITOR) {
+		tfw_http_msg_free((TfwHttpMsg *)req);
+		return;
+	}
+
 	if (reply) {
 		TfwCliConn *cli_conn = (TfwCliConn *)req->conn;
 		tfw_connection_unlink_msg(req->conn);
```

Comment:
Not visible here, but curly braces are required here in the `else` branch.

Reply:
Corrected.